pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -220,19 +220,20 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 	}
 
 	if cephCluster.Spec.External.Enable {
-		logger.Debug("external subvolume group creation is not supported, create it manually, the controller will assume it's there")
+		logger.Debug("skip creating external subvolume in external mode, create it manually, the controller will assume it's there")
```

Comment:
where are we generating the clientProfile for the external SVG?

Reply:
ahh it was not pushed for both svg and rns, added now